src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -1725,7 +1725,8 @@ bool ExecutorWorker::handleFinishedJob(std::shared_ptr<AbstractJob> job, SyncOpP
     auto networkJob(std::dynamic_pointer_cast<AbstractNetworkJob>(job));
     if (const bool isInconsistencyIssue = job->exitCause() == ExitCauseInvalidName;
         job->exitCode() == ExitCodeBackError && details::isManagedBackError(job->exitCause())) {
-        return handleManagedBackError(job->exitCause(), syncOp, isInconsistencyIssue, networkJob->isDownloadImpossible());
+        return handleManagedBackError(job->exitCause(), syncOp, isInconsistencyIssue,
```

Comment:
It will not be a trivial task as there are currently no tests for ExecutorWorker (except for utility functions). Since this pull request needs to be delivered in the 3.6.4 release, I have added this matter to the issue regarding another problem with this function that needs to be addressed before the 3.6.5 release.

Reply:
Unless someone has enough time before the release, but personally I won't as I am leaving in 20 minutes. 🌴

Reply:
https://infomaniak.atlassian.net/browse/KDESKTOP-1117?focusedCommentId=72139